install_config/master_node_configuration.adoc

```diff
@@ -1148,11 +1148,39 @@ AUDIT: id="5c3b8227-4af9-4322-8a71-542231c3887b" ip="127.0.0.1" method="GET" use
 AUDIT: id="5c3b8227-4af9-4322-8a71-542231c3887b" response="200"
 ----
 
-To enable and configure audit, edit the *_/etc/origin/master/master-config.yaml_*
-file or use a command similar to the following:
+[[proc-master-node-config-audit-config]]
 
+==== Enable Basic Auditing
+
+The following procedure enables basic auditing post installation.
+
+[NOTE]
+====
+Advanced Audit must be enabled during installation.
+====
+
+. To enable basic auditing, add the following variable declaration to your inventory file. Adjust values as appropriate.
```

Comment:
`Adding inventory file` and `Edit the *_/etc/origin/master/master-config.yaml_ ` are not in sequence. We could either `Adding inventory file` at installation time *Or* `Edit the *_/etc/origin/master/master-config.yaml_ ` post the installation.

Reply:
@gpei Would removing the "Adding the inventory file" step be sufficient in this case?

Reply:
maybe you can refer to the bug description directly:

```
Enable Basic Auditing POST INSTALLATION (Note ADVANCED audit must be enabled at installation time).

1. Edit /etc/origin/master/master-config.yaml file on all master nodes, as per the following example master-config.yaml file ...

auditConfig:
  auditFilePath: "/var/log/origin/audit-ocp.log"
  enabled: true
  maximumFileRetentionDays: 14
  maximumFileSizeMegabytes: 500
  maximumRetainedFiles: 15

2. Restart the API pods in your cluster

# /usr/local/bin/master-restart  api

Enable Basic Auditing at installation time

1. Add the following variable declaration to your inventory file. Adjust values as appropriate.

openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/log/oscp-audit/oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5}
```